browser-test/src/admin/admin_program_migration.test.ts

```diff
@@ -175,15 +177,16 @@ test.describe('program migration', () => {
       )
     })
 
+    await seedProgramsAndCategories(page)
+    await page.goto('/')
+    await adminPrograms.goToExportProgramPage(
+      'Comprehensive Sample Program',
+      'DRAFT',
+    )
+    let downloadedComprehensiveProgram =
```

Comment:
I moved this out of the `test.step` so we could access `downloadedComprehensiveProgram` in multiple test steps.